OpenRA.Platforms.Default/Sdl2PlatformWindow.cs

```diff
@@ -229,14 +233,14 @@ public GLProfile[] SupportedGLProfiles
 
 				var windowFlags = SDL.SDL_WindowFlags.SDL_WINDOW_OPENGL | SDL.SDL_WindowFlags.SDL_WINDOW_ALLOW_HIGHDPI;
 
-				// HiDPI doesn't work properly on OSX with (legacy) fullscreen mode
+				// HiDPI doesn't work properly on macOS with (legacy) fullscreen mode
 				if (Platform.CurrentPlatform == PlatformType.OSX && windowMode == WindowMode.Fullscreen)
 					SDL.SDL_SetHint(SDL.SDL_HINT_VIDEO_HIGHDPI_DISABLED, "1");
 
 				window = SDL.SDL_CreateWindow("OpenRA", SDL.SDL_WINDOWPOS_CENTERED_DISPLAY(videoDisplay), SDL.SDL_WINDOWPOS_CENTERED_DISPLAY(videoDisplay),
 					windowSize.Width, windowSize.Height, windowFlags);
 
-				if (Platform.CurrentPlatform == PlatformType.Linux)
+				if (Platform.CurrentPlatform == PlatformType.Linux && videoDriver != "wayland")
```

Comment:
```suggestion
				if (Platform.CurrentPlatform == PlatformType.Linux && !UseDeviceIndependentPixels
```

Why not use the boolean like above?

Reply:
The code in this block has nothing to do with display resolution/scaling so IMO it would be inappropriate to use that variable even though it does behave the same in this case.